cmd/watcher/main.go

```diff
@@ -58,6 +59,7 @@ var (
 	authToken               = flag.String("token", "", "Authentication token to use in requests. If not specified, on-cluster configuration is assumed.")
 	completedRunGracePeriod = flag.Duration("completed_run_grace_period", 0, "Grace period duration before Runs should be deleted. If 0, Runs will not be deleted. If < 0, Runs will be deleted immediately.")
 	threadiness             = flag.Int("threadiness", controller.DefaultThreadsPerController, "Number of threads (Go routines) allocated to each controller")
+	logsAPI                 = flag.Bool("logs_api", true, "Disable sending logs. If not set, the logs will be sent only if server support API for it")
```

Comment:
We are likely going to need to revisit this logic in the future. I like the fact that watcher will only forward logs if the API server publishes the service for it. There will be future cases (ex - logs forwarded via fluentd/fluentbit) where the watcher won't directly forward data, but will need to publish information as to where the logs landed.

OK to keep this here for now.